Update to Refresh Token Default value

SamCosby opened the pull request:
The default refresh value used to be 14, not 90. It appears that this has either changed or it's been misconfigured on this page. Updated it to be 14 unless otherwise.

azurecla commented:
Hi __@saschagottfried__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

KundanaP commented:
Thanks for the contribution. This looks good to merge.

SamCosby committed:
Update to Refresh Token Default value

The default refresh value used to be 14, not 90. It appears that this has either changed or it's been misconfigured on this page. Updated it to be 14 unless otherwise.
